lsr_role2collection.py

```diff
@@ -687,8 +687,12 @@ def cleanup_symlinks(path, role, rmlist):
             ):
                 node.rmdir()
         roles_dir = path / "roles"
-        if roles_dir.exists() and not any(roles_dir.iterdir()):
-            roles_dir.rmdir()
+        if roles_dir.exists():
+            for sr in roles_dir.iterdir():
+                if sr.is_symlink():
+                    sr.unlink()
+            if not any(roles_dir.iterdir()):
+                roles_dir.rmdir()
```

Comment:
Is this only being done in the `tests/roles/` directory?

Reply:
Yes, that's my intention.

As shown [here](https://src.osci.redhat.com/rpms/rhel-system-roles/pull-request/39#comment-18390), there are a few exceptions. E.g., in `tests`, these symlinks should be kept.
```
./ansible_collections/redhat/rhel_system_roles/tests/network/playbooks/files
./ansible_collections/redhat/rhel_system_roles/tests/network/playbooks/tasks
```
In `roles`, symlinks in vars are necessary.